lib/lib-common/package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@grpc/grpc-js": "^1.8.0",
     "rimraf": "^3.0.2",
-    "topcoder-interface": "github:topcoder-platform/plat-interface-definition#v0.0.4",
+    "topcoder-interface": "/Users/hamidtavakoli/topcoder/plat-interface-definition",
```

Comment:
Can we update this to use feature branch from git?

Reply:
Yup :)